docs/service_spec.md

```diff
@@ -102,11 +102,23 @@ v4 of the event schema originally required a `contextKeys` property on all featu
 
 This means that the SDK supports technology migrations, a feature which allows customers to migrate between data sources using well-defined migration stages.
 
-#### Capability `"persistent-data-store"`
+#### Capability `"persistent-data-store-{integration}"`
 
-This means the SDK is capable of interacting with external persistent data stores. The test harness must further identify which store types are supported through additional capabilities listed below.
+This means the SDK is capable of interacting with external persistent data stores. Each capability defines the specific persistence technology the SDK is capable of supporting. The full list of supported integrations is as follows:
 
-- `persistent-data-store-redis`: This means the SDK is capable of interacting with a Redis data store.
+- `consul`
+- `dynamodb`
+- `redis`
+
+The persistence store tests are unique in that they rely on external services to be available prior to running the test harness. Because of this, the test harness will not run these tests by default, even if the capabilities exist, without also provided the `-enable-persistence-tests` flag.
+
+These services can be easily provided through the use of docker. Example commands are shown below for your convenience.
+
+```shell
+docker run --network=host amazon/dynamodb-local
```

Comment:
For local tests you should be able to run these with network isolation? Not that most people aren't behind a NAT.

Reply:
I think `--network host` is experimental for mac and windows docker. Versus the normal port mapping.

```
The host networking driver only works on Linux hosts, but is available as a [beta feature](https://docs.docker.com/release-lifecycle/#beta) on Docker Desktop version 4.29 and later for Mac, Windows, and Linux. To enable this feature, navigate to the Resources tab in Settings, and then under Network select Enable host networking.
```

So, in theory:
`docker run -it -p 8000:8000 amazon/dynamodb-local`

Works on mac/windows.